scss/_patterns_image.scss

```diff
@@ -1,6 +1,38 @@
 @import 'settings';
 
 @mixin vf-p-image {
+  %vf-image-container--common {
+    img {
+      background-size: contain;
+    }
+  }
+
+  .p-image-container,
+  [class^='p-image-container--'] {
+    @extend %vf-image-container--common;
+  }
+
+  [class^='p-image-container'].is-highlighted {
+    background: rgba(0, 0, 0, 0.05);
```

Comment:
This would need to come from variable, possibly themed one. So we beed a value for both light and dark theme.

@lyubomir-popov What would be the dark version?

Reply:
pls use the variable called background-alt